Packs/CommonScripts/Scripts/GenericPollingScheduledTask/GenericPollingScheduledTask.js

```diff
@@ -224,7 +233,7 @@ function genericPollingScheduledTaskWithGuid() {
 
 function main() {
     res = getDemistoVersion();
-    platform = res.platform
+    platform = res.platform;
```

Comment:
according to [this](https://jira-hq.paloaltonetworks.local/browse/CRTX-81018?focusedCommentId=6836419&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-6836419) comment, line 242 should be:
```
    if ((buildNumber !== "REPLACE_THIS_WITH_CI_BUILD_NUM") && ((platform === "x2") && (compareVersions(version, MINIMUM_XSIAM_VERSION) >= 0)) && (parseInt(buildNumber) >= MINIMUM_BUILD_NUMBER))) {
```
also `MINIMUM_XSIAM_VERSION` should be `8.2.0`

similar in `ScheduleGenericPolling` script